roles/app/templates/project/managed.py

```diff
@@ -75,6 +77,14 @@ def INSTALLED_APPS(self):  # noqa
         'memory': '999m',
         'time': 3600,
     }
+    {% if SENTRY_DSN|string|length  %}
+
+    import os
+    import raven
+    RAVEN_CONFIG = {
+        'dsn': '{{ SENTRY_DSN }}',
```

Comment:
I usually add this so we have the git hash of the running version:
```
        'release': raven.fetch_git_sha(ROOT()),
```

Reply:
Fixed